### PR TITLE
gst-plugins-bad.rb fixed for Mavericks

### PR DESCRIPTION
**Pull request by tomoprime**
apple media now working for Mavericks

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by ilovezfs** [comment]
Do you know when/where (i.e., in which commit) this was fixed upstream?

**Comment by tomoprime** [comment]
It was working in 10.1.2 commit 5c4e9f2. I haven't looked beyond that.

File: Formula/gst-plugins-bad.rb [hunk 1]
```diff
@@ -48,13 +48,13 @@ def install
...
+    # UPDATE Apple Media support works in 1.10.2 for Mavericks.
     # upstream does not support Apple video for older SDKs
     # error: use of undeclared identifier 'AVQueuedSampleBufferRenderingStatusFailed'
     # https://github.com/Homebrew/legacy-homebrew/pull/35284
-    if MacOS.version <= :mavericks
-      args << "--disable-apple_media"
-    end
+    # if MacOS.version <= :mavericks
+    #   args << "--disable-apple_media"
+    # end
```
Comment (MikeMcQuaid) [inline Formula/gst-plugins-bad.rb]: Rather than commenting this out please remove all these lines (including your new comment). Thanks!

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks for your first  contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format is `gst-plugins-bad: apple media fixed on 10.9`. Please complete the issue template in future PRs where this format is detailed in the linked [Contributing guidelines](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md). Thanks!

**Pull request merged**